ql/termstructures/yield/oisratehelper.cpp

```diff
@@ -55,7 +55,8 @@ namespace QuantLib {
         //    be assigned a curve later; use a RelinkableHandle here
         swap_ = MakeOIS(tenor_, clonedOvernightIndex, 0.0)
             .withDiscountingTermStructure(discountRelinkableHandle_)
-            .withSettlementDays(settlementDays_);
+            .withSettlementDays(settlementDays_)
+            .withTelescopicValueDates(true);
```

Comment:
Same here --- may you take this as a constructor parameter?

Reply:
yes sure, shall the parameter be defaulted? Does the default depend on the switch?

Reply:
I'm not so sure about the switch anymore. I'd add this couple of default parameters so that the library works as before, but the behavior can be overridden by passing the non-default values.

Reply:
understood, makes sense